src/web/src/styles/telescope-post-content.css

```diff
@@ -16,11 +16,33 @@
   margin-left: auto;
   margin-right: auto;
   max-width: 100%;
+  max-height: 80vh;
   padding-top: 1.5rem;
   padding-bottom: 1rem;
   display: block;
   height: auto;
 }
+
+.telescope-post-content div > a > img {
+  width: 32px;
+  display: inline;
+  vertical-align: middle;
+}
+
+.telescope-post-content h1 > a > img {
```

Comment:
You could group these like:
```css
.telescope-post-content div, h1 > a > img {
...
}
```

Reply:
I tried this just now and it seems to ignore the CSS completely (as if it ain't there), I don't know why - thoughts?